internal/cloud/test.go

```diff
@@ -204,6 +208,7 @@ func (runner *TestSuiteRunner) Test() (moduletest.Status, tfdiags.Diagnostics) {
 		Filters:       runner.Filters,
 		TestDirectory: tfe.String(runner.TestingDirectory),
 		Verbose:       tfe.Bool(runner.Verbose),
+		Parallelism:   tfe.Int(runner.OperationParallelism),
```

Comment:
Just want to sanity check, that this being 0 will be overridden with a default at the API level?

Reply:
When the value is not given, i.e when the API calls the CLI, we use a default value. That override is done [here](https://github.com/hashicorp/terraform/blob/a58a31b9989eac05eab57bc09a02a3e092019b97/internal/command/arguments/test.go#L62), not at the API level